pkg/services/live/live.go

```diff
@@ -257,22 +258,30 @@ func (g *GrafanaLive) initChannel(id ChannelIdentifier) (models.ChannelHandler,
 	if id.Scope == "grafana" {
 		p, ok := g.GrafanaScope.Features[id.Namespace]
 		if ok {
-			return p.GetHandlerForPath(id.Path)
+			return p.GetHandlerForPath(id.Path, g.Publish)
 		}
 		return nil, fmt.Errorf("Unknown feature: %s", id.Namespace)
 	}
 
 	if id.Scope == "ds" {
-		return nil, fmt.Errorf("todo... look up datasource: %s", id.Namespace)
+		if id.Scope == "ds" {
```

Comment:
This condition is exactly the same as the above. Did you mean something else?

Reply:
Good catch, sloppy copy/paste on my part